2015-Fall/adrianbrink/Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by aevitas on 01/09/15.
+//  Copyright (c) 2015 aevitas. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    @IBOutlet weak var display: UILabel!
+    
+    var userIsTyping: Bool = false
+    var lastAction: Bool?
+    var lastValue = 0
+    var newValue = 0
+    
+    @IBAction func appendDigit(sender: UIButton) {
+        let digit = sender.currentTitle!
+        if userIsTyping {
+            display.text = display.text! + digit
+        }
+        else {
+            display.text = digit
+            userIsTyping = true
+        }
+    }
+
+    @IBAction func clearDisplay() {
+        display.text = "0"
+        lastValue = 0
+        lastAction = false
+        userIsTyping = false
+    }
+    
+    @IBAction func operators(sender: UIButton) {
+        let currentSymbol = sender.currentTitle!
+        var lastSymbol = ""
+        
+        if (lastAction == true) {
+            calculation(lastValue, symbol: lastSymbol)
+        } else {
+            lastValue = (display.text! as NSString).integerValue
```

Comment:
Consider using `.toInt()` instead. This returns an optional integer (`Int?`) which is `nil` if the string could not be converted to a number.